### ISSUE TITLE
fix translations for jst and js files

### ISSUE THREAD

**Issue by mrblueblue** [mrblueblue/gettext-loader#7]

**Comment by mrblueblue** [comment]
Gettext extractor must take into account nodes like the one below.

I believe the problem file is `isFunction` from the ast-utils

```
{
        "type": "ExpressionStatement",
        "expression": {
            "type": "CallExpression",
            "callee": {
                "type": "MemberExpression",
                "computed": false,
                "object": {
                    "type": "Identifier",
                    "name": "jed",
                    "range": [
                        27,
                        30
                    ],
                    "loc": {
                        "start": {
                            "line": 3,
                            "column": 0
                        },
                        "end": {
                            "line": 3,
                            "column": 3
                        }
                    }
                },
                "property": {
                    "type": "Identifier",
                    "name": "__",
                    "range": [
                        31,
                        33
                    ],
                    "loc": {
                        "start": {
                            "line": 3,
                            "column": 4
                        },
                        "end": {
                            "line": 3,
                            "column": 6
                        }
                    }
                },
                "range": [
                    27,
                    33
                ],
                "loc": {
                    "start": {
                        "line": 3,
                        "column": 0
                    },
                    "end": {
                        "line": 3,
                        "column": 6
                    }
                }
            },
            "arguments": [
                {
                    "type": "Literal",
                    "value": "a simple example",
                    "raw": "'a simple example'",
                    "range": [
                        34,
                        52
                    ],
                    "loc": {
                        "start": {
                            "line": 3,
                            "column": 7
                        },
                        "end": {
                            "line": 3,
                            "column": 25
                        }
                    }
                }
            ],
            "range": [
                27,
                53
            ],
            "loc": {
                "start": {
                    "line": 3,
                    "column": 0
                },
                "end": {
                    "line": 3,
                    "column": 26
                }
            }
        },
        "range": [
            27,
            53
        ],
        "loc": {
            "start": {
                "line": 3,
                "column": 0
            },
            "end": {
                "line": 3,
                "column": 26
            }
        }
    },
```

**Comment by mrblueblue** [comment]
fix in version 0.2.0
